### DISCUSSION TITLE
It seems like the plugin has been hacked

### DISCUSSION THREAD

**Discussion by rpeck** [qunash/chatgpt-advanced#230] [General]
For the first time in a month or so I logged into ChatGPT's web interface this morning. I found several chats that I did not do. They clearly were done with templates for the prompts. Disturbingly, some of them were based on web searches and other history from my browser, but not all were. 

One of them had WebChatGPT as its subject, so I strongly suspect that the plugin has been hacked. I've uninstalled it.

Thoughts?

**Comment by zamoth22** [comment]
this was a great extension to be honest but now it simply stopped working
